providers/os/id/networki/routes.go

```diff
@@ -0,0 +1,115 @@
+// Copyright (c) Mondoo, Inc.
```

Comment:
I think all the new files should be moved into https://github.com/mondoohq/cnquery/tree/main/providers/os/resources/networkinterface. We do not need to move existing files from https://github.com/mondoohq/cnquery/tree/main/providers/os/id/networki as port of this PR, but we should not add things to the package when we already know things should move into the resource package